src/components/Breadcrumb/Breadcrumb.vue

```diff
@@ -0,0 +1,157 @@
+<!--
+ - @copyright Copyright (c) 2020 Raimund Schlüßler <raimund.schluessler@mailbox.org>
+ -
+ - @author Raimund Schlüßler <raimund.schluessler@mailbox.org>
+ -
+ - @license GNU AGPL version 3 or any later version
+ -
+ - This program is free software: you can redistribute it and/or modify
+ - it under the terms of the GNU Affero General Public License as
+ - published by the Free Software Foundation, either version 3 of the
+ - License, or (at your option) any later version.
+ -
+ - This program is distributed in the hope that it will be useful,
+ - but WITHOUT ANY WARRANTY; without even the implied warranty of
+ - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ - GNU Affero General Public License for more details.
+ -
+ - You should have received a copy of the GNU Affero General Public License
+ - along with this program. If not, see <http://www.gnu.org/licenses/>.
+ -
+ -->
+
+<docs>
+
+### General description
+
+This component is meant to be used inside a Breadcrumbs component.
+
+</docs>
+
+<template>
+	<div ref="crumb"
+		class="crumb"
+		:class="{over: hovering}"
+		draggable="false"
+		@dragstart.prevent="() => {}"
+		@drop.prevent="dropped"
+		@dragover.prevent="() => {}"
+		@dragenter="dragEnter"
+		@dragleave="dragLeave">
+		<element :is="tag" :to="to" :href="href">
+			<span v-if="icon" :class="icon" class="icon" />
+			<span v-else>{{ title }}</span>
+		</element>
+		<slot />
```

Comment:
See https://github.com/nextcloud/nextcloud-vue/pull/913#discussion_r399481447

```suggestion
		<Actions>
			<slot />
		</Actions>
```

Of course spacing needs to be fixed a bit when we have one action:
![Capture d’écran_2020-03-28_14-08-24](https://user-images.githubusercontent.com/14975046/77823733-9bbb4680-70fd-11ea-852e-b1404dc06d04.png)

Reply:
Also like we do in the Sidebar we should maybe forward the :force-menu="forceMenu" prop :)

Reply:
Will do that.

Reply:
Done.